packaging/python_wheel/build_docker_image.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
conditionally as part of the Jenkins plan. The build has a parameter the user can switch to 'on', 'off' by default.